docs/development/security/gdpr_regulation.md

```diff
@@ -0,0 +1,156 @@
+        * A task being carried out for public interest.
+
+### Children specific regulations
+
+If the child is under the age of 16, it is only legal to collected and process data if a parent with custody of that child, gives the permission to do so. It falls under the responsibility of the data-responsible to ensure with reasonable effort that this is done. The individual countries might enact national regulation that puts this age down to 13 years.
+
+### Controller and Processor
+
+The data controller might be a company that seeks to collect and process personal information, while a data processor might be another company that provides a cloud-service that the data controller company wishes to use. 
+* Data protection principles and measures must be designed intro the business processes for products and services. This includes measures such as pseudonymisation and high level of privacy by default (Article 25). This is the responsibility of the data controller, even if processing is carried out by a data processor. The data controller keeps internal record of the processing of personal data. This will give an overview of the treatments of the data that is initiated. Such an overview is a necessary prerequisite to fulfill a number of obligations, such as: considerations on what information to process; handling insights requests and review of breaches of personal data security. This is an internal document that data protection agency can request at any time. 
+    * A report ([ENISA on privacy and data protection by design](https://www.enisa.europa.eu/publications/privacy-and-data-protection-by-design)) specifies what needs to be done to achieve privacy and data protection by default. For instance, encryption and decryption operations must be carried out locally, and not be a remote service as the keys and data must remain in the power of the data owner.
+* Data protection impact assessment have to be conducted when specific risks occur to the rights and freedoms of data subjects. Prior approval of the data protection authorities is required for high risks (Article 35).
+* Pseudonymisation: is a required process for stored data that transforms personal data in such a way that the resulting data cannot be attributed to a specific data subject without the use of additional information. An example of such, is encryption that renders the original data unintelligible and the process cannot be reversed without access to the correct decryption key.
+* Records of processing activities: must be maintained, and include purposes of the processing, categories involved and envisaged time limits.
+* Security of personal data. In the case of a breach, is the data controller is under a legal obligation to notify the supervisory authority within 72 hours of becoming aware of it. Further does the data subject(s) have to be informed if there is a high risk of an adverse impact.
```

Comment:
I think there are a bit of inconsistency here. In the bullet above, you use semi colon to determine the "title" of that bullet point, here you use a period sign. I think just decide on one or the other.